rozy/src/installers/conda.rs

```diff
@@ -28,6 +28,7 @@ pub fn conda_install(
     let conda_cache_dir = tempdir()?;
     let mut command = std::process::Command::new(conda_bin);
     command.env("CONDA_PKGS_DIRS", conda_cache_dir.path());
+    command.stdout(Into::<std::process::Stdio>::into(os_pipe::dup_stderr()?));
```

Comment:
Does  `Stdio::from(os_pipe...)` not work? I don't really know rust all too well, just saw that in some docs.

Reply:
Ah, I think that should work. They are related traits: https://doc.rust-lang.org/rust-by-example/conversion/from_into.html